macros/macros.go

```diff
@@ -8,6 +8,7 @@ import (
 // EndpointTemplateParams specifies params for an endpoint template
 type EndpointTemplateParams struct {
 	Host        string
+	Pubid       string
```

Comment:
Pubid is string.

Reply:
Okay. Is it just a string of numbers or it could be a string of characters or mix of characters and numbers too? Even if that's the case, it might still be easier to have just `PublisherID` and make that a string rather than have `PubID` and `PublisherID` both. We can easily change an integer publisher ID to a string in that case.

Reply:
Ah actually I see from your example json that `PubID` could be a mix of characters and numbers. In that case as suggested above, could you please rather change `PublisherID` to type string itself and the integer `PublisherID` values used before can be converted into string as needed?

Reply:
@trchandraprakash pinging to see if you have had a chance to make the above change

Reply:
@mansinahar I am OOO this week. Will update coming monday. Also if I change PublisherID macro to be a string, won't it impact other bidder adapters who are considering it as integer.

Reply:
Well, yeah those values will have to be converted into string before setting them to `PublisherID` and this PR would have to do that work. From a quick look hopefully it shouldn't be a lot of changes though